scss/navbar.scss

```diff
@@ -0,0 +1,21 @@
+.nav {
+  align-items: center;
+  box-shadow: 0 0 10px $black;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  margin-bottom: 50px;
```

Comment:
this won't be necessary when you change the position to fixed because the component starts working in a separate layer, you should have this in the layout:
```
<body>
  <header ... </header>
  <div class="content">...</content>
  <footer>...</footer>
</body>
```
set margins in the `.content` element and that's it.